### PR TITLE
Fixed attested tls sample document broken link

### PR DESCRIPTION
**Pull request by yentsanglee**
Fixed attested tls sample document broken link.

Signed-off-by: Yen Lee <yenlee@microsoft.com>

### CODE REVIEW

**Review by qiucwang — COMMENTED** [review]
Thanks for fixing it.
LGTM.

**Review by qiucwang — APPROVED** [review]

File: samples/attested_tls/AttestedTLSREADME.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ There are two types of Attested TLS connections:
 
   - [Transport Layer Security (TLS)](https://en.wikipedia.org/wiki/Transport_Layer_Security) a cryptographic protocol designed to provide communications security over a computer network.
 
-  - [Open Enclave Remote Attestation](https://github.com/openenclave/openenclave/tree/master/samples/remote_attestation#what-is-attestation): Remote Attestation is the concept of a HW entity or of a
+  - [Open Enclave Remote Attestation](https://github.com/openenclave/openenclave/tree/master/samples/attestation#what-is-attestation): Remote Attestation is the concept of a HW entity or of a
```
Comment (shnwc) [inline samples/attested_tls/AttestedTLSREADME.md:27]: The attestation sample works for both local and remote attestation. Does the Attested TLS sample work for both types attestation? If so how about drop  all reference of "remote" from the text?

Comment (yentsanglee) [reply samples/attested_tls/AttestedTLSREADME.md:27]: Attested TLS targets on remote attestation.

**Review by shnwc — APPROVED** [review]

**Pull request merged**